src/main/java/org/ow2/proactive/catalog/repository/entity/CatalogObjectRevisionEntity.java

```diff
@@ -68,11 +69,13 @@
                                                                                                "COMMIT_TIME" }), indexes = { @Index(name = "REVISION_INDEX", columnList = "BUCKET,NAME,COMMIT_TIME") })
 public class CatalogObjectRevisionEntity implements Comparable, Serializable {
 
-    @GeneratedValue(generator = "uuid2")
-    @GenericGenerator(name = "uuid2", strategy = "uuid2")
-    @Column(name = "ID", columnDefinition = "BINARY(16)")
     @Id
-    private UUID id;
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "CATALOG_OBJECT_REVISION_SEQUENCE")
+    @GenericGenerator(name = "CATALOG_OBJECT_REVISION_SEQUENCE", strategy = "org.hibernate.id.enhanced.SequenceStyleGenerator", parameters = { @Parameter(name = "sequence_name", value = "CATALOG_OBJECT_REVISION_SEQUENCE"),
+                                                                                                                                               @Parameter(name = "initial_value", value = "1"),
+                                                                                                                                               @Parameter(name = "increment_size", value = "1") })
```

Comment:
Is the space between the annotations on purpose?

Reply:
it is done by spotlessApply plugin